db-engine/src/main/java/com/splicemachine/db/iapi/types/SQLTime.java

```diff
@@ -1180,7 +1180,7 @@ public void read(Row row, int ordinal) throws StandardException {
 			setToNull();
 		else {
 			isNull = false;
-			encodedTime = computeEncodedTime(row.getDate(ordinal));
+			encodedTime = computeEncodedTime(row.getTimestamp(ordinal));
```

Comment:
Time translates to timestamp in Spark world.  I screwed up and had it as a date.